src/AutorestSwift/Models/Complex/CodeModel.swift

```diff
@@ -43,12 +43,12 @@ class CodeModel: Codable, LanguageShortcut {
     }
 
     /// Lookup a global parameter by name.
-    func globalParameter(for name: String) -> Parameter? {
-        return globalParameters?.first { $0.name == name }
+    func globalParameter(for name: String) -> ParameterType? {
+        return globalParameters?.first(named: name)
     }
 
     func getApiVersion() -> String {
-        if let constantSchema = globalParameter(for: "apiVersion")?.schema as? ConstantSchema {
+        if let constantSchema = globalParameter(for: "api-version")?.schema as? ConstantSchema {
```

Comment:
the '-' is added to 'api-versoin'. Will that break anything?

Reply:
Without it, it wasn't generating the API names correctly. They were showing up just a "v".

Reply:
So, according to David:
```
Modelerfour normalizes api version parameters for you, you shouldn't have to do anything special.  In this case, they either need to rename it in the swagger to api-version or use x-ms-api-version: true
```